srcs/player.c

```diff
@@ -34,7 +40,6 @@ void
 		info->player.angle += 2 * M_PI;
 	if (2 * M_PI < info->player.angle)
 		info->player.angle -= 2 * M_PI;
-	info->player.dx = cos(info->player.angle);
-	info->player.dy = sin(info->player.angle);
+	angle_delta(info->player.angle, &(info->player.dx), &(info->player.dy));
 	print_minimap(info);
```

Comment:
The player prints the map?

Reply:
Yes, when the player moves, it updates the player's position and then prints the mini map again.